index.html

```diff
@@ -11,54 +11,153 @@
 </head>
 
 <body>
-    <header>
-        <nav>
-            <div class="logo">
-                <a href="#">Jonah.</a>
-            </div>
-            <div class="mobile-menu">
-                <a href="#"><i class="fas fa-bars"></i></a>
-            </div>
-        </nav>
-    </header>
-    
     <div class="container">
         <main>
-            <section class="headline">
-                I'm Jonah<br>Glad to see you!
-            </section>
-
-            <section class="body">
-                <p>
-                    I’m a software developer! I can help you build a product feature or website. Look through some
-                    of my work and experience! If you like what you see and have a project you need coded, don’t
-                    hesitate to contact me.
-                </p>
-            </section>
-
-            <section class="social-media">
-                <p>LET'S CONNECT</p>
+            <section class="works">
+                <div class="card">
+                    <div class="snapshot"></div>
+                    <div class="project-title">Tonic</div>
+                    <div class="project-subtitle">
+                        <div class="flex-container-tonic">
+                            <div class="canopy">CANOPY</div>
+                            <div class="dot"></div>
+                            <div>Back End Dev</div>
+                            <div class="dot"></div>
+                            <div>2015</div>
+                        </div>
+                    </div>
+                    <div class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                    <div class="categories">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javaScript</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
```

Comment:
- [ ] According to the requirements, You should Implement the button interactions (enable, hover, etc.). Kindly, add the button interactions based on your [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 👌.

![1](https://user-images.githubusercontent.com/45231276/176833382-1d1d826d-6c66-47ac-bec8-36427809918f.PNG)

![image](https://user-images.githubusercontent.com/45231276/176984389-0354a1db-f777-4cd4-9317-9d9928a9bc90.png)

- [ ] Note: you should do this on all buttons👌